src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -200,22 +209,33 @@ public void updateFilteredListToShowFloating() {
     @Override
     public void updateFilteredListToShowOverdue() {
         filteredTasks.setPredicate((Predicate<? super ReadOnlyTask>) task -> {
-            return isOverdue(task);
+            return isOverdue(task) && !(task.isDone());
         });
         indicateViewListChanged(ViewCommand.TYPE_OVERDUE);
     }
 
     //@@author A0135998H
     @Override
+    public void updateFilteredListToShowPending() {
```

Comment:
Btw just saying these methods can be improved and consolidated as one method only possibly